flaskbb/message/views.py

```diff
@@ -87,7 +87,7 @@ def get(self, conversation_id):
         form = self.form()
         return render_template("message/conversation.html", conversation=conversation, form=form)
 
-    @requires_message_box_space
+    #@requires_message_box_space
```

Comment:
I'd rather see this decorator modified than commented out.

Reply:
By “modified”, you mean it should be deleted. Or implement another method that checks for the total number messages?

Reply:
Probably not deleted, but changed so it only looks at sent messages rather than received if that's what we agree is proper.

However, most boards I've been a member of that enforce PM counts do count both, so I'm not convinced of this PR in general. @sh4nks may disagree and I'll defer to him if he does.

Reply:
So it’s more like the alternative 2 that I proposed in #337? Right now it’s counting **conversations** (with the exception that you can’t answer once you reach the quota), but it should actually count **messages**?